.devcontainer/centos7-bld.dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ghcr.io/smanders/buildpro/centos7-bld:21.13
+LABEL maintainer="smanders"
+LABEL org.opencontainers.image.source https://github.com/smanders/buildpro
+SHELL ["/bin/bash", "-c"]
+USER 0
+# build docker image offline support
+ARG ADDSRC1 ADDSRC2
+ADD ${ADDSRC1} ${ADDSRC2} ${EXTERN_DIR}/
+# CRTool
+ARG CRTOOL
```

Comment:
You can toss CRTool, pluginSDK, internpro, webpro

Reply:
I commented all that stuff out and then I got warnings or something from the base image they point to in buildpro, because I wasn't setting those arguments.